client/src/components/title.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import RoomInfo from './roomInfo.jsx';
+
+function Title(props) {
+  if (props.guestno > 1) {
```

Comment:
You can use the ternary operator here to make your statements more concise ( and it's the main way to do if-else conditional rendering in React!)

Reply:
Refactored to use ternary operator.